app/src/main/java/org/mozilla/focus/activity/MainActivity.java

```diff
@@ -463,6 +480,41 @@ private void dismissAllMenus() {
         }
     }
 
+    OnLongClickListener onLongClickListener = new OnLongClickListener() {
+        @Override
+        public boolean onLongClick(View v) {
+            menu.cancel();
+            switch (v.getId()) {
+                case R.id.menu_night_mode:
+                    final Settings settings = Settings.getInstance(MainActivity.this);
```

Comment:
can we just call onNightModeClicked() in MainActivity?

Reply:
Agreed, I cleaned up the flow in onNightModeClicked() to reduce duplicate code.